official/vision/beta/projects/yolo/modeling/backbones/darknet.py

```diff
@@ -666,7 +665,8 @@ def build_darknet(
     l2_regularizer: tf.keras.regularizers.Regularizer = None) -> tf.keras.Model:
   """Builds darknet."""
 
-  backbone_cfg = backbone_config.get()
+  backbone_cfg = backbone_config.backbone.get()
```

Comment:
it should be backbone_config.get(), right?

Reply:
addressed